src/groovy/org/pih/warehouse/api/Putaway.groovy

```diff
@@ -34,6 +35,15 @@ class Putaway {
         orderedBy(nullable:true)
     }
 
+    List<PutawayItem> getPutawayItems() {
+       return putawayItems.sort { a,b ->
+           sortBy ? a."${sortBy}" <=> b."${sortBy}"  ?:
```

Comment:
I think this should be && and not ?
`(sortBy && a."${sortBy}" <=> b."${sortBy}")  ?:`